src/Common/tests/System/Collections/DictionaryExtensions.cs

```diff
@@ -1,20 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-namespace System.Collections.Generic
-{
-    internal static class DictionaryExtensions
```

Comment:
The reason I removed this is because was an ambiguous call with the new `Collection.ExtensionsTryAdd`. I'll investigate resolving the problem re: desktop.